### ISSUE TITLE
Update Crystal Kemal results with build --release

### ISSUE THREAD

**Issue by sdogruyol** [reu/fibaas#3]
Hey @reu 

Thanks for doing this repository. I've checked the results and wondered that if the Kemal application was built with `build --release`?

Building the Kemal application with release flag and optimizations nearly doubles the performance (62147.08 rps with 1.03ms average) compared to your results. (on a 2013 Macbook Retina Pro with 8G ram)

Here's how you can reproduce

```
crystal build --release fibaas.cr
./fibass
```

Could you please rerun and update the results. Thanks!

**Comment by reu** [comment]
Oh, hey @sdogruyol! I am not quite sure if I build it with the release flag, but I will do that and will report back!

Thanks a lot for the tip!

**Comment by sdogruyol** [comment]
Meanwhile this has ben done with Crystal `0.15.0`

**Comment by reu** [comment]
Hey @sdogruyol, sorry for the long delay...

I tried to re-run the benchmark with the `--release` flag you mentioned (actually, I did run with that the first time), but I just can't reach the same numbers as yours. Maybe I did something wrong while installing Crystal (I have to confess, Crystal and Elixir are the languages that I've never used before doing this project).

I installed Crystal via Homebrew, and the version I have installed here is this one: 

``` bash
$ crystal -v
Crystal 0.15.0 (Fri Apr  1 17:57:09 UTC 2016)
```

I will soon run these tests on a beefy AWS instance, maybe I will get better results there.
